### PR TITLE
{bio}[GCC/13.3.0] BEDTools v2.31.1, BamTools v2.5.2

### PR DESCRIPTION
**Pull request by maxim-masterov**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/2aa707d7750fdc6fd0e174bcf6556010 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    'bin',
+    'docs',
+    'data',
+    'genomes',
+    'scripts',
+    'test',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'bin/pairToBed', 'bin/mergeBed', 'bin/bedToBam', 'bin/fastaFromBed'],
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb:47]: @maxim-masterov I think the previous version was clearer with the `for x in` etc.

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    'dirs': ['bin', 'docs', 'data', 'genomes', 'scripts', 'test'],
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb:48]: Same here, I preferred the use of `files_to_copy`.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb (hunk 1 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb]: my bad, changed it back

File: easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb (hunk 2 above)
Comment (maxim-masterov) [reply easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.1-GCC-13.3.0.eb]: done

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-el8.cloud.in.bmrc.ox.ac.uk - Linux Rocky Linux 8.10, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/bc90e6617a761a62494e10534551c41f for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @maxim-masterov!

**Pull request merged**